code/lib/channel-postmessage/src/index.ts

```diff
@@ -36,8 +36,8 @@ export class PostmsgTransport {
   constructor(private readonly config: Config) {
     this.buffer = [];
     this.handler = null;
-    if (globalWindow) {
-      globalWindow.addEventListener('message', this.handleEvent.bind(this), false);
+    if (global) {
```

Comment:
Why is the check for `global` needed here?

Reply:
Well it was checking `global.window` before. Which of course.. in the browser is just `globalThis.globalThis`.
In node environment `global.window` would be `undefined` except if the environment was enriched by `jsdom`.

How about we change this to do proper feature detection?

```ts
    if (typeof global?.addEventListener === 'function') {
      global.addEventListener('message', this.handleEvent.bind(this), false);
    }
```